src/veh_interact.cpp

```diff
@@ -1418,9 +1418,9 @@ bool veh_interact::do_tirechange( std::string &msg )
             return false;
 
         case LACK_TOOLS:
-            msg = string_format( _( "To change a wheel you need a <color_%1$s>wrench</color> and either "
-                                    "<color_%2$s>lifting equipment</color> or <color_%3$s>%4$d</color> strength." ),
-                                 status_color( has_wrench ), status_color( has_jack ),
+            msg = string_format( _( "To change a wheel you need a <color_%1$s>wrench</color>, a <color_%2$s>wheel</color> and either "
```

Comment:
Nitpick, but [prefer Oxford comma](https://github.com/CleverRaven/Cataclysm-DDA/blob/67668987fee36729cac0c24d9d7c99e8a29063a8/doc/MANUAL_OF_STYLE.md#L9) after "wheel":

`a <color_%2$s>wheel</color>, and either`

Reply:
Fair enough, grammar was never my strong point :) Just need to wait for my latest build to finish.